### PR TITLE
Replace omz_downloader/converter to native download/mo

### PR DESCRIPTION
**Pull request by sbalandi**

### CODE REVIEW

File: notebooks/215-image-inpainting/215-image-inpainting.ipynb [hunk 1]
```diff
@@ -34,8 +37,8 @@
...
    "source": [
-    "### Download the Model\n",
-    "Models can be downloaded with `omz_downloader` - a command-line tool for downloading models from the Open Model Zoo. `gmcnn-places2-tf` is the omz name for the considered model. You can find the names of all available models [here](https://github.com/openvinotoolkit/open_model_zoo/blob/master/models/public/index.md) and [here](https://github.com/openvinotoolkit/open_model_zoo/blob/master/models/intel/index.md). The selected model comes from the public directory, which means it must be converted into Intermediate Representation (IR). This step is skipped if the model is already downloaded.\n"
+    "### Download and Unzip the Model\n",
+    "`gmcnn-places2-tf` is the omz name for the considered model. You can find the names of all available models [here](https://github.com/openvinotoolkit/open_model_zoo/blob/master/models/public/index.md) and [here](https://github.com/openvinotoolkit/open_model_zoo/blob/master/models/intel/index.md). The selected model comes from the public directory, which means it must be converted into Intermediate Representation (IR). This step will be skipped if the model is already downloaded.\n"
```
Comment (eaidova) [inline notebooks/215-image-inpainting/215-image-inpainting.ipynb:41]: not sure that it is good idea to tell users how to download models from our s3 and provide information where other models can be found. Can we do not highlight model source in description? (just imagine that it is public model, but we store it on our server for download efficiency)

**Comment by sbalandi** [comment]
Hi, @eaidova I tried way with downloading original model source and applying patch, but you was right about version and freez_model fail with tensorflow error related to using function from lower version

**Review by adrianboguszewski — APPROVED** [review]

File: notebooks/215-image-inpainting/215-image-inpainting.ipynb (hunk 1 above)
Comment (sbalandi) [reply notebooks/215-image-inpainting/215-image-inpainting.ipynb]: done

File: notebooks/215-image-inpainting/215-image-inpainting.ipynb [hunk 2]
```diff
@@ -35,7 +38,7 @@
    "metadata": {},
    "source": [
     "### Download the Model\n",
-    "Models can be downloaded with `omz_downloader` - a command-line tool for downloading models from the Open Model Zoo. `gmcnn-places2-tf` is the omz name for the considered model. You can find the names of all available models [here](https://github.com/openvinotoolkit/open_model_zoo/blob/master/models/public/index.md) and [here](https://github.com/openvinotoolkit/open_model_zoo/blob/master/models/intel/index.md). The selected model comes from the public directory, which means it must be converted into Intermediate Representation (IR). This step is skipped if the model is already downloaded.\n"
+    "Donwload model with omz name `gmcnn-places2-tf`(this step will be skipped if the model is already downloaded) and then unzip it. \n"
```
Comment (eaidova) [inline notebooks/215-image-inpainting/215-image-inpainting.ipynb:41]:
```suggestion
    "Download `gmcnn-places2-tf`model (this step will be skipped if the model is already downloaded) and then unzip it. Downloaded model stored in TensorFlow frozen graph format. The steps how this frozen graph can be obtained from original model checkpoint can be found in this [instruction](https://docs.openvino.ai/latest/omz_models_model_gmcnn_places2_tf.html#steps-to-reproduce-conversion-to-frozen-graph)\n"
```
and please also add link to instruction how to convert original checkpoint to frozen graph ()

File: notebooks/215-image-inpainting/215-image-inpainting.ipynb [hunk 3]
```diff
@@ -77,19 +81,17 @@
...
+    "# Run Model Optimizer to convert model to OpenVINO IR FP32 format, if the IR file does not exist.\n",
     "if not ir_path.exists():\n",
-    "    print(\"Exporting TensorFlow model to IR... This may take a few minutes.\")\n",
-    "    convert_command = f\"omz_converter \" \\\n",
-    "                      f\"--name {model_name} \" \\\n",
-    "                      f\"--download_dir {base_model_dir} \" \\\n",
-    "                      f\"--precisions {precision}\"\n",
-    "    ! $convert_command\n",
+    "    mo_command = f'mo --input_model {model_path} --input Placeholder,Placeholder_1 --input_shape [1,512,680,3],[1,512,680,1] --output=Minimum --output_dir {model_dir}'\n",
+    "    display(Markdown(f\"Model Optimizer command to convert the ONNX model to IR: `{mo_command}`\"))\n",
```
Comment (eaidova) [inline notebooks/215-image-inpainting/215-image-inpainting.ipynb:90]: it is tensorflow, not ONNX

File: notebooks/215-image-inpainting/215-image-inpainting.ipynb [hunk 4]
```diff
@@ -77,19 +81,17 @@
...
-    "# Run Model Optimizer if the OpenVINO IR model file does not exist.\n",
+    "# Run Model Optimizer to convert model to OpenVINO IR FP32 format, if the IR file does not exist.\n",
     "if not ir_path.exists():\n",
-    "    print(\"Exporting TensorFlow model to IR... This may take a few minutes.\")\n",
-    "    convert_command = f\"omz_converter \" \\\n",
-    "                      f\"--name {model_name} \" \\\n",
-    "                      f\"--download_dir {base_model_dir} \" \\\n",
-    "                      f\"--precisions {precision}\"\n",
-    "    ! $convert_command\n",
+    "    mo_command = f'mo --input_model {model_path} --input Placeholder,Placeholder_1 --input_shape [1,512,680,3],[1,512,680,1] --output=Minimum --output_dir {model_dir}'\n",
```
Comment (eaidova) [inline notebooks/215-image-inpainting/215-image-inpainting.ipynb:89]: Can we try to convert this model without specification --input, --output and --input_shape?

Comment (sbalandi) [reply notebooks/215-image-inpainting/215-image-inpainting.ipynb:89]: I tried and on step:
`N, H, W, C = input_layer.shape`
got error:
```
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-6-02fb957c3494> in <module>
----> 1 N, H, W, C = input_layer.shape

ValueError: get_shape was called on a descriptor::Tensor with dynamic shape
```

File: notebooks/215-image-inpainting/215-image-inpainting.ipynb (hunk 2 above)
Comment (sbalandi) [reply notebooks/215-image-inpainting/215-image-inpainting.ipynb:41]: done

File: notebooks/215-image-inpainting/215-image-inpainting.ipynb (hunk 3 above)
Comment (sbalandi) [reply notebooks/215-image-inpainting/215-image-inpainting.ipynb:90]: done

File: notebooks/215-image-inpainting/215-image-inpainting.ipynb (hunk 4 above)
Comment (eaidova) [reply notebooks/215-image-inpainting/215-image-inpainting.ipynb:89]: pk, then let's stay as is for now

**Review by eaidova — APPROVED** [review]

**Pull request merged**